builtin/mainmenu/content/pkgmgr.lua

```diff
@@ -755,11 +755,21 @@ end
 
 --------------------------------------------------------------------------------
 function pkgmgr.find_by_gameid(gameid)
+	if not gameid then
+		return nil, nil
+	end
 	for i, game in ipairs(pkgmgr.games) do
 		if game.id == gameid then
 			return game, i
 		end
 	end
+	for i, game in ipairs(pkgmgr.games) do
+		for _, alias in pairs(game.aliases) do
+			if pkgmgr.normalize_game_id(alias) == pkgmgr.normalize_game_id(gameid) then
+				return game, i
+			end
```

Comment:
What if multiple games claim to be an alias of the previous one? Looking at the Mineclone, Mineclonia and whatnot forks.

Reply:
The first found is used I guess...? What else could you even do?

Reply:
Just thinking loud. Perhaps a warning that multiple candidates were found? Aside from that I don't see any way to determine which would be the saner pick.

Reply:
Added a warning.